lib/ihci/data/facility_blocks_list.csv

```diff
@@ -0,0 +1,601 @@
+state,facility,district,block,,,

```

Comment:
OK for this info to be checked into the repo?

Reply:
Should be ok, this information is publicly accessible. This does make the name of facilities public. Will check.